libs/dh/admin/feature-permissions/src/index.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export * from './lib/overview/dh-admin-permission-overview.component';
```

Comment:
nitpick: rename component, selector and related files to `dh-admin-permissions-overview` ("permission" is plural).

Reply:
![image](https://user-images.githubusercontent.com/1250254/222711171-aaf75da7-f16c-41d9-967e-ebc0fbc64cde.png)

Reply:
Good point. A bit like "fish" -> "fish"/"fishes"

In that case it's fine with either. You can resolve if you prefer "permission". :)